www/src/components/LinkedHeading.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 export interface ILinkedHeading {
   h: string,
-  children: React.ReactNode,
-  id: string,
+  children?: React.ReactNode,
```

Comment:
[clarification] Why is `children`, and `id` considered optional in this interface, when the prop types previously denoted them as required? I'm not sure why we'd have a use case for rendering `LinkedHeading` _without_ `children` (a heading without content?). Similarly, the `id` is needed for heading to be skip-linkable; if `id` is not required, wouldn't that largely differ from the intent of this `LinkedHeading` component?